internal/command/image/show.go

```diff
@@ -91,11 +134,104 @@ func runShow(ctx context.Context) error {
 		},
 	}
 
-	return render.VerticalTable(io.Out, "Deployment Status", obj,
+	return render.VerticalTable(io.Out, "Image Details", obj,
+		"Registry",
+		"Repository",
+		"Tag",
+		"Version",
+		"Digest",
+	)
+}
+
+func showMachineImage(ctx context.Context, app *api.App) error {
+	var (
+		client = client.FromContext(ctx).API()
+		io     = iostreams.FromContext(ctx)
+	)
+
+	// if we have machine_id as an arg, we want to show the image for that machine only
+	if len(flag.Args(ctx)) > 0 {
+		flaps, err := flaps.New(ctx, app)
```

Comment:
I'm thinking you need to rebase from latest? `flaps.New` expect the second parameter to be of time `*api.AppCompact` so I'm not entirely sure how this is compiling.

Reply:
@jipperinbham just rebased it